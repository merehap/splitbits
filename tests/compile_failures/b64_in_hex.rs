use splitbits::*;

fn main() {
    let a = 0b11;
    let b = true;
    let c = false;
    let d = 0b01;
    let e = 0b1111_0000;
    combinehex!("ddAB cCbb bDEF aaH1 XY45 6789 eeee eeee");
}
