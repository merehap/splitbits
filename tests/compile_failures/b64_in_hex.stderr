error: proc macro panicked
 --> tests/compile_failures/b64_in_hex.rs:9:5
  |
9 |     combinehex!("ddAB cCbb bDEF aaH1 XY45 6789 eeee eeee");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Invalid characters for base 16 detected: ['H', 'X', 'Y']. Did you mean to use a higher base?
